treeherder/log_parser/artifactbuilders.pyx

```diff
@@ -86,7 +96,7 @@ class BuildbotLogViewArtifactBuilder(ArtifactBuilderBase):
             HeaderParser(),
             StepParser(check_errors=check_errors)
         ]
-        self.name = "Structured Log"
+        self.name = "text_log_summary"
```

Comment:
Yeah, I've addressed that in this PR now: https://github.com/mozilla/treeherder-ui/pull/399 (added it last night).  Adding the handling for both names now in ``webapp/api/artifact.py`` - ``ArtifactViewSet.list``